packages/x-data-grid-premium/src/tests/clipboard.DataGridPremium.test.tsx

```diff
@@ -6,12 +6,14 @@ import {
   DataGridPremiumProps,
   GridColDef,
 } from '@mui/x-data-grid-premium';
-// @ts-ignore Remove once the test utils are typed
 import { createRenderer, fireEvent, userEvent, waitFor } from '@mui/internal-test-utils';
 import { expect } from 'chai';
-import { stub, SinonStub, spy } from 'sinon';
+import { SinonSpy, spy } from 'sinon';
 import { getCell, getColumnValues, sleep } from 'test/utils/helperFn';
 import { getBasicGridData } from '@mui/x-data-grid-generator';
+import rtlUserEvent from '@testing-library/user-event';
```

Comment:
I tested https://github.com/mui/material-ui/pull/42978 locally and it seems to work without problems. 👌 

```diff
-import { createRenderer, fireEvent, userEvent, waitFor } from '@mui/internal-test-utils';
+import { createRenderer, fireEvent, userEvent, waitFor, rtlUserEvent } from '@mui/internal-test-utils';
-import rtlUserEvent from '@testing-library/user-event';
```